civis/KYC/kyc_311_to_esri.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[12]:
```

Comment:
Can you remove these `In[]` prompts? They don't really add much useful info in this context.

Reply:
removed.